packages/lisk-p2p/src/peer.ts

```diff
@@ -713,6 +713,6 @@ export const connectAndFetchPeerInfo = async (
 
 		return { peerInfo, socket };
 	} catch (error) {
-		return error;
+		throw error;
```

Comment:
Since the error is just being re-thrown as-is and we're not doing anything else with it here, we don't need the try-catch block in this function.

Reply:
I think you commented on the outdated code, but now I'm throwing an error but capturing peer ip and port for better logging

Reply:
It appears that you already fixed this in a later commit so ignore.